code/modules/spacepods/spacepod.dm

```diff
@@ -73,7 +73,7 @@
 	pr_int_temp_processor = new /datum/global_iterator/pod_preserve_temp(list(src))
 	pr_give_air = new /datum/global_iterator/pod_tank_give_air(list(src))
 	equipment_system = new(src)
-	equipment_system.installed_modules = new/list()
+	equipment_system.installed_modules = list()
```

Comment:
I did test the difference on a local server, and it seemed to work without issue. It's merely defining the variable set. Up until I defined it here, it was merely a reference variable. My guess is it's merely changing the type of variable to a list, and thus doesn't have to create a new list.